packages/server/lib/plugins/index.js

```diff
@@ -37,6 +38,43 @@ const registerHandler = (handler) => {
   handlers.push(handler)
 }
 
+const getChildOptions = (config) => {
+  const childOptions = {
+    stdio: 'pipe',
+    env: {
+      ...process.env,
+      NODE_OPTIONS: process.env.ORIGINAL_NODE_OPTIONS || '',
+    },
+  }
+
+  if (config.resolvedNodePath) {
+    debug('launching using custom node version %o', _.pick(config, ['resolvedNodePath', 'resolvedNodeVersion']))
+    childOptions.execPath = config.resolvedNodePath
+  }
+
+  // https://github.com/cypress-io/cypress/issues/18914
+  // If we're on node version 17 or higher, we need the
+  // NODE_ENV --openssl-legacy-provider so that webpack can continue to use
+  // the md4 hash function. This would cause an error prior to node 17
+  // though, so we have to detect node's major version before spawning the
+  // plugins process.
+
+  // To be removed on update to webpack >= 5.61, which no longer relies on
+  // node's builtin crypto.hash function.
+  if (semver.satisfies(config.resolvedNodeVersion, '>=17.0.0')) {
+    childOptions.env.NODE_OPTIONS += ' --openssl-legacy-provider'
```

Comment:
is it possible for users to bring their own version of webpack?

Reply:
If so, could we check to see if they're using our version or theirs before adding the option?

Reply:
actually i'm not sure that really matters, this is probably fine

Reply:
I believe that while they can bring their own webpack *config*, but not version. And if I'm wrong and they can, it should still be harmless to add the arg.